pytest failure

Just run `pytest` in the system environment fails as 'thank-you-stars' hasnt been installed.  That can be avoid by creating a `__main__.py` and using `python -m ..`
```py
_______________________________________________________________________________ Test_tys.test_normal[command0-0] _______________________________________________________________________________

self = <test_cli.Test_tys object at 0x7f7756f6a610>, command = ['thank-you-stars', 'pytablewriter', '--dry-run'], expected = 0

    @pytest.mark.parametrize(
        ["command", "expected"],
        [
            [[TYS_CMD, "pytablewriter", DRY_RUN_OPT], 0],
            [[TYS_CMD, "thank-you-stars", CHK_OPT], 0],
            [[TYS_CMD, "thank-you-stars", CHK_OPT, "--include-owner-repo", DEPTH_OPT, 3], 0],
            [[TYS_CMD, "pytablewriter", DRY_RUN_OPT, DEPTH_OPT, -1], errno.EINVAL],
        ],
    )
    def test_normal(self, command, expected):
>       assert SubprocessRunner(command).run() == expected

test/test_cli.py:32: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/usr/lib/python3.8/site-packages/subprocrunner/_subprocess_runner.py:133: in run
    self.__verify_command()
/usr/lib/python3.8/site-packages/subprocrunner/_subprocess_runner.py:246: in __verify_command
    Which(base_command).verify()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = command=thank-you-stars, is_exist=False, follow_symlinks=False

    def verify(self) -> None:
        if not self.is_exist():
>           raise CommandError(
                "command not found: '{}'".format(self.command), cmd=self.command, errno=errno.ENOENT
            )
E           subprocrunner.error.CommandError: command not found: 'thank-you-stars'

/usr/lib/python3.8/site-packages/subprocrunner/_which.py:43: CommandError
```

--- a/test/test_cli.py
+++ b/test/test_cli.py
@@ -3,12 +3,13 @@
 """
 
 import errno
+import sys
 
 import pytest
 from subprocrunner import SubprocessRunner
 
 
-TYS_CMD = ["python", "-m", "thank_you_stars"]
+TYS_CMD = [sys.executable, "-m", "thank_you_stars"]
 CHK_OPT = "--check"
 DEPTH_OPT = "--depth"
 DRY_RUN_OPT = "--dry-run"

--- a/thank_you_stars/__main__.py
+++ b/thank_you_stars/__main__.py
@@ -163,7 +163,7 @@ def extract_package_name(options):
         return options.target
 
     if os.path.isfile("setup.py"):
-        runner = SubprocessRunner(["python", "setup.py", "--name"])
+        runner = SubprocessRunner([sys.executable, "setup.py", "--name"])
         if runner.run() == 0:
             return runner.stdout.strip().lower()
 

--- a/thank_you_stars/_extractor.py
+++ b/thank_you_stars/_extractor.py
@@ -1,4 +1,5 @@
 import re
+import sys
 from collections import namedtuple
 from difflib import SequenceMatcher
 
@@ -162,7 +163,7 @@ class GithubStarredInfoExtractor:
 
     @staticmethod
     def __normalize_pkg_name(name):
-        return re.sub("python", "", name, flags=re.IGNORECASE).lower()
+        return re.sub(sys.executable, "", name, flags=re.IGNORECASE).lower()
 
     def __fetch_pypi_info(self, pypi_pkg_name):
         cache_filepath = self.__pypi_cache_mgr.get_pkg_cache_filepath(pypi_pkg_name, "pypi_desc")

